src/genie/libs/parser/iosxr/show_isis.py

```diff
@@ -5134,3 +5136,99 @@ def get_value(value):
                 continue
 
         return result_dict
+
+################################################################################
+
+"""Schema for 'show isis segment-routing srv6 locators'"""
+class ShowIsisSegmentRoutingSrv6LocatorsSchema(MetaParser):
+    schema = {
+        'instance': {
+            Any(): {
+                'locators': {
+                    Any(): {
+                        'id': int,
+                        'algo': int,
+                        'prefix': str,
+                        'status': str
+                    },
+                }
+            }
+        }
+    }
+
+# ==============================================
+# Parser for 'show isis segment-routing srv6 locators'
+# ==============================================
+
+class ShowIsisSegmentRoutingSrv6Locators(ShowIsisSegmentRoutingSrv6LocatorsSchema):
+    """Parser for:
+    * show isis segment-routing srv6 locators
+    * show isis instance {instance} segment-routing srv6 locators
+    """
+    cli_command = ['show isis segment-routing srv6 locators',
+                   'show isis instance {instance} segment-routing srv6 locators']
+
+    """
+    IS-IS 1 SRv6 Locators
+    Name                  ID       Algo  Prefix                    Status
+    ------                ----     ----  ------                    ------
+    ALGO_0                1        0     cafe:0:100::/48           Active
+    ALGO_128              2        128   cafe:0:128::/48           Active
+    ALGO_129              3        129   cafe:0:129::/48           Active
+    """
```

Comment:
Since we have this output stored in the `ShowIsisSegmentRoutingSrv6Locators/cli/equal/golden_output_output.txt` test file, we can remove it from here and save some space

Reply:
Updated in latest commit